README.md

```diff
@@ -38,6 +43,17 @@ cider log add 'Support for rocket fuel and kerosene'
 cider log fix 'No more wheels falling off'
 ```
 
+## Setting the project version
+```
+cider set <version>
```

Comment:
My initial vision was to combine this with the existing `version` command. E.g.
- `cider version` prints the version
- `cider version 1.2.3` sets the version

What do you think?

Reply:
It's ok)
You seem to have support for command aliases.